giphyAPI.py

```diff
@@ -0,0 +1,3 @@
+class GiphyAPI:
+    def function(): #spaceholder
```

Comment:
I don't think function is a reserved word in Python, but you may want to actually add a real name for these functions because this code looks pretty confusing.